android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="uniffi.nimbus">
```

Comment:
This should be `org.mozilla.nimbus` but keeping it like this because the package name of the generated bindings is `uniffi.nimbus` by default